perma_web/api/views.py

```diff
@@ -692,6 +692,9 @@ def post(self, request, format=None):
         # mark batch with user
         if not request.user.is_authenticated:
             raise PermissionDenied()
+        if request.content_type != 'application/json':
```

Comment:
This could be a function that can be used elsewhere, but not sure what routes also need to have this error. All of them?